entries/n/noridianmedicareportal.com.json

```
@@ -0,0 +1,16 @@
+{
+  "Noridian Medicare": {
+    "domain": "noridianmedicareportal.com",
```

```suggestion
    "domain": "noridianmedicareportal.com",
    "url": "https://www.noridianmedicareportal.com",
```

Because going to `https://noridianmedicareportal.com` doesn't work, the `url` key is needed.